poi-5.2.2/pom.xml

```diff
@@ -258,34 +259,44 @@
                                 <excludes>
                                     <exclude>*:*:*:sources</exclude>
                                     <exclude>${project.groupId}:${project.artifactId}:*:sources</exclude>
+                                    <exclude>org.apache.logging.log4j:*</exclude>
                                 </excludes>
                             </artifactSet>
                             <filters>
-                                <filter>
+                               <filter>
                                     <artifact>${pkgGroupId}:poi</artifact>
-                                    <excludes>
-                                        <exclude>**</exclude>
-                                    </excludes>
-                                </filter>
+                                    <includes>
+                                        <include>META-INF/services/**</include>
+                                    </includes>
+                               </filter>
                                 <filter>
                                     <artifact>${pkgGroupId}:poi-scratchpad</artifact>
-                                    <excludes>
-                                        <exclude>**</exclude>
-                                    </excludes>
+                                    <includes>
+                                        <include>META-INF/services/**</include>
+                                    </includes>
                                 </filter>
                                 <filter>
                                     <artifact>${pkgGroupId}:poi-ooxml</artifact>
-                                    <excludes>
-                                        <exclude>**</exclude>
-                                    </excludes>
+                                    <includes>
+                                        <include>META-INF/services/**</include>
+                                    </includes>
+                                </filter>
+								<filter>
+                                    <artifact>${pkgGroupId}:poi-ooxml-lite</artifact>
+                                    <includes>
+                                        <include>META-INF/services/**</include>
+                                    </includes>
                                 </filter>
                                 <filter>
                                     <artifact>com.github.virtuald:curvesapi</artifact>
-                                    <excludes>
-                                        <exclude>**</exclude>
-                                    </excludes>
+                                    <includes>
```

Comment:
It's possible to define this in the private package statement of the maven-bundle-plugin too.

Reply:
and the maven-bundle-plugin can merge the services like ServicesResourceTransformer ?